Source/DiabloUI/errorart.cpp

```diff
@@ -2,7 +2,7 @@
 
 namespace devilution {
 
-const unsigned char btnData[] = {
+const std::uint8_t btnData[] = {

```

Comment:
Nitpick: this is not a BYTE -> uint8 conversion

Reply:
It is part of the same cleanup.